cluster-autoscaler/FAQ.md

```diff
@@ -759,6 +759,7 @@ The following startup parameters are supported for cluster autoscaler:
 | `daemonset-eviction-for-empty-nodes` | Whether DaemonSet pods will be gracefully terminated from empty nodes | false
 | `daemonset-eviction-for-occupied-nodes` | Whether DaemonSet pods will be gracefully terminated from non-empty nodes | true
 | `feature-gates` | A set of key=value pairs that describe feature gates for alpha/experimental features. | ""
+| `enable-event-duplication` | Enable the autoscaler to print duplicated events within a 5 minute window. | false
```

Comment:
The name of the flag is weird. What do you think about "enable-event-deduplicaiton" (with default set to true) or "record-duplicated-events" (with default set to false).

Reply:
It'd be also good to provide some context on when enable/disable is recommended https://github.com/kubernetes/autoscaler/issues/2495#issuecomment-547981551 cc @vivekbagade @losipiuk

Reply:
yeah, the name is a bit odd. i wanted to preserve the default behavior we have now and allow the flag to enable the old behavior. i think `record-duplicated-events` sounds like a good idea to me.